core/src/main/java/org/geysermc/geyser/util/SettingsUtils.java

```diff
@@ -75,6 +76,10 @@ public static CustomForm buildForm(GeyserSession session) {
             if (session.getGeyser().getConfig().isAllowCustomSkulls()) {
                 builder.toggle("geyser.settings.option.customSkulls", session.getPreferencesCache().isPrefersCustomSkulls());
             }
+
+            if (session.getGeyser().getConfig().isAboveBedrockNetherBuilding()) {
+                builder.toggle("geyser.settings.option.aboveBedrockNetherBuilding", session.getPreferencesCache().isPrefersAboveBedrockNetherBuilding());
```

Comment:
This needs adding to the GeyserMC/languages repository.

Reply:
You can make a PR to https://github.com/GeyserMC/languages .